pkg/operator/sync.go

```diff
@@ -325,11 +325,17 @@ func (optr *Operator) syncMachineConfigPools(config *renderConfig) error {
 	return nil
 }
 
-func (optr *Operator) syncMachineConfigController(config *renderConfig) error {
-	for _, path := range []string{
-		"manifests/machineconfigcontroller/clusterrole.yaml",
-		"manifests/machineconfigcontroller/events-clusterrole.yaml",
-	} {
+type manifestPaths struct {
+	clusterRoles        []string
+	roleBindings        []string
+	clusterRoleBindings []string
+	serviceAccounts     []string
+	secrets             []string
+	daemonsets          []string
+}
+
+func (optr *Operator) syncManifests(config *renderConfig, paths manifestPaths) error {
```

Comment:
This isn't a true sync but a helper function called by other functions probably rename this.

Error wise is it going to be clear which caller sync caused it?

Reply:
I'll make the wrapping errors different for each function

Do you have a suggestion for a better name? `applyManifests`?

Reply:
wfm!